admin/webUI/index.html

```diff
@@ -140,6 +140,7 @@
     </script>
 
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://use.fontawesome.com/9ee605636b.js"></script>
```

Comment:
Don't need to use Font Awesome icons, above that line I have included Material Icons link. Refer https://material.io/icons/ to find relevant icon.
For sort arrow icon check this : https://material.io/icons/#ic_arrow_drop_down